packages/bridge-ui/src/App.svelte

```diff
@@ -146,31 +146,44 @@
 
       const apiTxs = await $relayerApi.GetAllByAddress(userAddress);
 
+      console.log(apiTxs);
+
       const blockInfoMap = await $relayerApi.GetBlockInfo();
       relayerBlockInfoMap.set(blockInfoMap);
 
       const txs = await $transactioner.GetAllByAddress(userAddress);
 
+      console.log(txs);
```

Comment:
do we want to keep these console.log's? i don't mind, but sometimes it's a lint rule `no-console-log`. same with below.

Reply:
will remove!